app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -273,9 +273,17 @@ def raw_connect_v4(server, port, path, username, password, service, scheme = 'ht
       # Get the timeout from the configuration:
       timeout, = ems_timeouts(:ems_redhat, service)
 
+      # Build the URL:
```

Comment:
comment seems pretty useless as it's clear you're calling build on a URI object.